src/schema.ts

```diff
@@ -120,6 +121,12 @@ export function build(data: any,  tableSchema: TableSchema<TableSchemaFieldDescr
       vlType = VLType.NOMINAL;
     }
 
+    if (vlType === VLType.NOMINAL 
+      && distinct / fieldProfile.count > DEFAULT_QUERY_CONFIG.minPercentUniqueForKey
```

Comment:
Reading directly from `DEFAULT_QUERY_CONFIG` here means that whatever config we provide wouldn't be used.  

We already call `opt = extend({}, DEFAULT_QUERY_CONFIG, opt);` at the beginning of this method, so better read the config from `opt`.